src/model/book.model.js

```diff
@@ -0,0 +1,22 @@
+// eslint-disable-next-line no-unused-vars
+class Book {
+  constructor(
+    { title, author, category, status, number, price, check } = { check: false }
+  ) {
+    this.id = this.generateID();
+    this.title = title;
+    this.author = author;
+    this.category = category;
+    this.status = status;
+    this.number = number;
+    this.price = price;
+    this.check = check;
+  }
+
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
Why do you add this line?

Reply:
![image](https://github.com/TITO-TRAINING/Duc_Hieu_Nguyen_JS_PRACTICE/assets/94243225/2c69dd33-718e-4cda-b871-9f2e24d83a75)
if i don't add this line, eslint will show an error